spec/controllers/catarse_moip/notifications_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe CatarseMoip::NotificationsController, type: :controller do
+
+  context "when the flow works fine" do
```

Comment:
You could put this context inside a describe block with the controller action, like:
```ruby
...
 describe "POST #create_notification" do
  context "when the flow works fine" do
   ...
  end
 end
...
```